docs/settings.md

```diff
@@ -74,6 +76,8 @@ Content-type: application/vnd.api+json
     "type": "io.cozy.settings",
     "id": "io.cozy.settings.disk-usage",
     "attributes": {
+      "is_limited": true,
+      "total_size": "123456789",
```

Comment:
And I'd have not put a `is_limited` field. If true, there is a `quota`/`total_size`. If false, we just omit this field.